app/assets/stylesheets/pages/track-page.scss

```diff
@@ -157,24 +158,23 @@ body.controller-my-tracks.action-show {
             background-image:image-url("code-sample-frame.png");
             background-size:contain;
             background-repeat:no-repeat;
-            padding:30px 10px;
+            padding:28px 6px;
             height:200px;
             width: 258px;
 
             pre {
-                background:#fafafa;
                 @include font-size(10, 13);
-                padding:2px 5px;
-                height:144px;
-                code {
-                    background:#f8f8f8;
-                }
+                padding:10px 10px;
+                height:151px;
+                margin:0;
+                border-radius: 0 0 9px 9px;
```

Comment:
As the dark background now cuts against the graphic, I've tidied the padding and border radiuses to make it fit neatly.